domain/actionSpace/urgentWishForChildren.unit.ts

```diff
@@ -6,17 +6,18 @@ import {
     Dwarf,
     EntityMutation,
     FurnishingId,
+    Game,
     isMutationOfType,
     Mutation,
     Player,
 } from "../entity";
 
 describe("Urgent Wish for Children", () => {
-    shouldPlaceDwarf(UrgentWishForChildren.execute);
+    shouldPlaceDwarf(UrgentWishForChildren.execute, initPlayerFonds);
 
     describe("should create and place a new dwarf", () => {
         it("should add new dwarf to player", function () {
-            const { game, player } = buildBaseObjects();
+            const { game, player } = buildUrgentWishBaseObjects();
```

Comment:
Actually, if we don't pass any custom argument to `buildUrgentWithBaseObjects()`, we could call it in a `beforeEach` block

And that way, we may also be able to change the `shouldPlaceDwarf` function to make it only return an `it` block that takes a Player and a Game in argument and that does bot checks. It may be simpler, wdyt?

Reply:
Let's do this, changed! https://github.com/BoredGamesWorkshop/caverna/pull/9/commits/a93116c9dca65d6d1d1af651ed8f1906a584e493
https://github.com/BoredGamesWorkshop/caverna/pull/9/commits/e94d22c23f496dadcd9b49ba6c2b222802bce5c0
https://github.com/BoredGamesWorkshop/caverna/pull/9/commits/1216d98a5b8337e8c20bf22c1641825bcc93cc7d

Reply:
Great!